packages/simple_trail/docs/README.md

```diff
@@ -0,0 +1,53 @@
+## What is SimpleTrail
+
+SimpleTrail makes it easy to keep history of attribute changes on a model.
+
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'simple_trail'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install simple_trail
+
+## Usage
+
+```ruby
+class Project < ApplicationRecord
+  include ::SimpleTrail::Recordable
+end
+```
+
+Now you can access the object history through `SimpleTrail.for(object)` like:
+
+```ruby
+project = Project.create
+SimpleTrail.for(project).size
+# => 1
+```
+
+The user performing the action will be recorded from the Thread local `:user_id`.
```

Comment:
I'm not sure what use case this is here. Will a user_id be recorded on a table? Where? How? What if I want something other than a user_id?